dotnet/src/Planners/Planners.Core/Sequential/SKContextSequentialPlannerExtensions.cs

```diff
@@ -82,23 +83,23 @@ public static class SKContextSequentialPlannerExtensions
             result = new List<FunctionView>();
 
             // Remember functions in memory so that they can be searched.
-            await RememberFunctionsAsync(context, config.Memory, availableFunctions, cancellationToken).ConfigureAwait(false);
+            await RememberFunctionsAsync(context, semanticMemoryConfig.Memory, availableFunctions, cancellationToken).ConfigureAwait(false);
 
             // Search for functions that match the semantic query.
-            var memories = config.Memory.SearchAsync(
+            var memories = semanticMemoryConfig.Memory.SearchAsync(
                 PlannerMemoryCollectionName,
                 semanticQuery!,
-                config.MaxRelevantFunctions,
-                config.RelevancyThreshold.Value,
-                cancellationToken: cancellationToken);
+                semanticMemoryConfig.MaxRelevantFunctions,
+                semanticMemoryConfig.RelevancyThreshold.HasValue ? semanticMemoryConfig.RelevancyThreshold.Value : 0.0,
```

Comment:
good call

Reply:
yeah I feel confident about this now that it's in a separate config object, so if you give anything there, that feels like a reasonable default given Memory is available.